### PR TITLE
Preparing for v1.0.0

### LINKED ISSUE #19: types: merge *_json files with regular files
For exemple, `Coordinates` struct have two associated files (coordinates.go and coordinates_json.go).
The _json file implement the `json.Unmarshaller`. We want to move this implementation to the `coordinates.go` file directly.

### PR DESCRIPTION
**Pull request by nicolasdscp**

### CODE REVIEW

File: types/coord.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+// This is the Go representation of "Coord". It implements Place.
+// See http://doc.navitia.io/#standard-objects.
+type Coord struct {
+	Longitude float64 `json:"lon"`
+	Latitude  float64 `json:"lat"`
+}
+
+// jsonCoord define the JSON implementation of Coord struct
+type jsonCoord struct {
+	Latitude  string `json:"lat"`
```
Comment (alc6) [inline types/coord.go:19]: eventhough it's not a time, its good to have units in the name of the variable, or at least in a comment (see https://github.com/uber-go/guide/blob/master/style.md#use-timetime-and-timeduration-with-external-systems)

Comment (nicolasdscp) [reply types/coord.go:19]: I'm agree, I'm gonna change it and fields name to match perfectly to the API. Here's a preview:

```go
type Coord struct {
	Lon float64 `json:"lon"` // Longitude in degrees
	Lat  float64 `json:"lat"`  // Latitude in degrees
}
```

**Comment by adelcasse** [comment]
What's the plan on this ?
We were using our own fork of @aabizri 's navitia library for internal use.
We might probably be interested in contributing ...

**Comment by nicolasdscp** [comment]
Hi @adelcasse a few days ago, we agreed with @aabizri to transfer the repo on our organization in order to continue the development of the lib which was no longer maintained. We are starting the update work and we are probably planning to release a new version. We are also interested to have external contributors.

**Comment by thomas-tacquet** [comment]
> 
> 
> What's the plan on this ?
> We were using our own fork of @aabizri 's navitia library for internal use.
> We might probably be interested in contributing ...

Hi @adelcasse 

We recenlty retrieved this repository in our `govitia` team to enhance it and to makes it fit all navitia features, we are currently working on contributing guide, documentation, cleaning and testing all files and working on a strict release policy to avoid breaking changes and match navitia's API releases.

A roadmap is being build and will be publicly available soon. Before releasing a v1.0.0 version we have lot of work to do.

If you have exceptations about new features or questions feel free to open issues or PR 👍

File: types/container.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package types
+ package types
```
Comment (thomas-tacquet) [inline types/container.go]: please ensure you have gofumpt installed for code formating

**Comment by adelcasse** [comment]
Hi @nicolasdscp, I've seen this (I had notifications enabled on changes to this repo).

Concerning the version number, it might be interesting to switch directly to "v2" instead of v1 to avoid breaking changes for those using the previous versions (I don't know if there are other than us, if it's just us it's not a big deal and not critical but we never know ;) ) : https://blog.golang.org/v2-go-modules

I'll follow the roadmap when you publish it and see with our development team what we could contribute.

File: types/coord.go (hunk 1 above)
Comment (nicolasdscp) [reply types/coord.go:19]: Changes has been reverted to avoid breaking changes

**Pull request merged**